loader/loader.go

```diff
@@ -1,11 +1,16 @@
 package loader
 
 import (
+	"bytes"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"strings"
+
+	"github.com/docker/cli/cli/compose/loader"
```

Comment:
Looks like this one should be grouped with the ones below

Reply:
Fixed 👍